### PR TITLE
TAB-8176: Java11/Jetty10/other-3rdParty-updates

### PR DESCRIPTION
**Pull request by GaryWKeim**

### CODE REVIEW

File: management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java [hunk 1]
```diff
@@ -97,17 +95,18 @@ public void start() throws Exception {
...
       ServerConnector connector;
       if (sslCtxt != null) {
         // A new HttpConfiguration object is needed for the next connector and you can pass the old one as an
         // argument to effectively clone the contents. On this HttpConfiguration object we add a
         // SecureRequestCustomizer which is how a new connector is able to resolve the https connection before
         // handing control over to the Jetty Server.
         HttpConfiguration httpsConfig = new HttpConfiguration(httpConfig);
-        httpsConfig.addCustomizer(new SecureRequestCustomizer());
+        SecureRequestCustomizer src = new SecureRequestCustomizer();
+        src.setSniHostCheck(false);
```
Comment (chrisdennis) [inline management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java:106]: Why false?

File: management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java [hunk 2]
```diff
@@ -97,17 +95,18 @@ public void start() throws Exception {
       HttpConfiguration httpConfig = new HttpConfiguration();
       httpConfig.setSecureScheme("https");
       httpConfig.setSecurePort(port);
-      httpConfig.setBlockingTimeout(0); // make it same as idleTime (defaults to 30s.)
```
Comment (chrisdennis) [inline management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java:100]: What changed here?

File: management-common-v2/pom.xml [hunk 3]
```diff
@@ -20,11 +20,11 @@
       <artifactId>management-core</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-servlet</artifactId>
-      <scope>provided</scope>
-    </dependency>
+<!--    <dependency>-->
```
Comment (chrisdennis) [inline management-common-v2/pom.xml:23]: Why not just deleted (this and the half dozen other cases)?

File: management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java (hunk 1 above)
Comment (GaryWKeim) [reply management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java:106]: Actually, I wanted to pass that in the SecureRequestCustomizer c'tor. We talked about this over MM.... security breaks if SNI checks are enabled and you're doing localhost stuff. Further, the TMC won't ever be deployed in a Virtual Hosts-like environment.

File: management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java (hunk 2 above)
Comment (GaryWKeim) [reply management-core-resources/src/main/java/org/terracotta/management/embedded/StandaloneServer.java:100]: That API was removed and replaced with HttpConfig.min{Request,Response}DataRate, which default to 0.

File: management-common-v2/pom.xml (hunk 3 above)
Comment (GaryWKeim) [reply management-common-v2/pom.xml:23]: I meant to.

**Review by chrisdennis — APPROVED** [review]

**Comment by chrisdennis** [comment]
I think you'll need to tweak the `azure-pipelines.yml` file to get the CI build running in Java 11.

**Comment by GaryWKeim** [comment]
I'm having trouble getting the check to pass... it seems to be using the old versions of forge-parent and thirdparty-BOM.

**Pull request merged**